test_scripts/API/VehicleData/commonVehicleData.lua

```diff
@@ -311,6 +311,28 @@ function m.ptUpdateMin(pTbl)
   pTbl.policy_table.vehicle_data = nil
 end
 
+function ternary(cond, a, b)
+  return cond and a or b
+end
+
+function empty(table)
```

Comment:
```suggestion
function isTableEmpty(table)
```
I would propose to rename function

Reply:
@VjKlepikov please check https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/15/commits/e86562fbe628dad0e5104d3c6551760077385116

Reply:
@VjKlepikov please check https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/15/commits/e86562fbe628dad0e5104d3c6551760077385116